plugin/pkg/admission/eventratelimit/apis/eventratelimit/types.go

```diff
@@ -41,25 +41,25 @@ const (
 // Configuration provides configuration for the EventRateLimit admission
```

Comment:
Actually we didn't take any precaution to prevent user from writing a config in the internal format, see [test](https://github.com/caesarxuchao/kubernetes/blob/bd18cebcf40d9ddf9f478b25feb0e44a0f387cd3/plugin/pkg/admission/eventratelimit/config_test.go). Thus, this PR is possible to break such users, though the such users are relying on unsupported behavior.

Similarly, can you double check if we ever deserialize/serialize the internal version of the resourcequota controller config?